docs/admin/templates/extending-templates/parameters.md

```diff
@@ -79,6 +79,34 @@ data "coder_parameter" "security_groups" {
 }
 ```
 
+> [!NOTE] Overriding a `list(string)` on the CLI is tricky because:
+>
+> - `--parameter "parameter_name=parameter_value"` is parsed as CSV.
+> - `parameter_value` is parsed as JSON.
+>
+> So, to properly specify a `list(string)` with the `--parameter` CLI argument,
+> you will need to take care of both CSV quoting and shell quoting.
+>
+> For the above example, to override the default values of the `security_groups`
+> parameter, you will need to pass the following argument to `coder create`:
+>
+> ```
+> --parameter "\"security_groups=[\"\"DevOps Security Group\"\",\"\"Backend Security Group\"\"]\""
+> ```
+>
+> You can use [this Go Playground link](https://go.dev/play/p/yvI9rdtS0ch) to
```

Comment:
Security question: is it safe to leave it as is? Do these links expire? Maybe just drop a snippet around docs.

Reply:
Although I do see it's not very 'user-friendly' to have to edit some Go code to figure something out. I'll move the playground link to the corresponding test instead but keep the above example invocation generated by the playground link.